docs/getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+## 7. Deploy with ChugSplash
+
+> Note: When deploying, upgrading, or testing your contracts locally, you must always use an Anvil node running as a stand-alone process.
+
+To create an Anvil node in a stand-alone process, run:
+
+```
+anvil
+```
+
+In another terminal window, run the following command to execute your deployment:
+
+```
+forge script script/MyFirstProject.s.sol
+```
+
+You should see the following output:
+```
+== Logs ==
+  Success!
+  HelloChugSplash: 0x...
+```
+
+You've deployed your first upgradeable contract locally!
+
+## 8. Test with ChugSplash
+
+In the folder that contains your Foundry tests (usually `test/`), create your test file. We'll call it `MyFirstProject.t.sol`.
+
+Inside, copy and paste the following:
+```sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import "forge-std/Test.sol";
+import "chugsplash-foundry/src/ChugSplash.sol";
+import "../src/HelloChugSplash.sol";
+
+contract ChugSplashTest is Test {
+    // Your upgradeable contract
+    HelloChugSplash helloChugSplash;
+
+    function setUp() public {
+        // Create a ChugSplash instance
+        ChugSplash chugsplash = new ChugSplash();
+
+        // Define the path from the project root to your ChugSplash file.
+        string memory chugsplashFilePath = "./chugsplash/hello-chugsplash.json";
+
+        // Deploy all contracts in your ChugSplash file (in this case, just HelloChugSplash.sol)
+        chugsplash.deploy(chugsplashFilePath, true);
+
+        // You *must* refresh EVM state after calling `chugsplash.deploy`.
+        chugsplash.refresh();
+
+        // Connect to the deployed contract
+        helloChugSplash = HelloChugSplash(chugsplash.getAddress(chugsplashFilePath, "HelloChugSplash"));
+    }
+
+    function testNumber() public {
+        assertEq(helloChugSplash.number(), 1);
+    }
+
+    function testStored() public {
+        assertEq(helloChugSplash.stored(), true);
+    }
+
+    function testStorageName() public {
+        assertEq(helloChugSplash.storageName(), "First");
+    }
+
+    function testOtherStorage() public {
+        assertEq(helloChugSplash.otherStorage(), 0x1111111111111111111111111111111111111111);
+    }
+}
+```
+
+Notice that you need to call `chugsplash.refresh()` after calling `chugsplash.deploy(...)`.
+
+Run your tests using the command:
+```
+forge test
+```
+
+## 9. Upgrade with ChugSplash
+
+Upgrades are defined in exactly the same format as deployments.
+
+To upgrade `HelloChugSplash`, navigate to your existing ChugSplash file, `hello-chugsplash.json`. Then, simply change the variables to new values. For example, change the `number` variable from `1` to `2`, or `storageName` from `"First"` to `"Second"`.
```

Comment:
This is a little confusing. It kind of implies that changing the variables is somehow necessary. I would instead provide of an example of adding a new variable to a contract since that's a bit closer to the typical upgrade use case.